### DISCUSSION TITLE
SX1262 BUSY always high after sleep mode

### DISCUSSION THREAD

**Discussion by phemidal** [Lora-net/LoRaMac-node#1321] [Q&A]
Hi,

I work on a custom board with a SX1262 transceiver and a STM32F429 MCU. I ported all functions for the MCU and tried to run the "periodic-uplink-lpp" example but it stucks in LmHandlerInit().
After the call to Radio.Sleep() in the function LoRaMacInitialization() the BUSY pin goes 500ns after the rising edge from NSS from low to high as it is stated in the datasheet. 
![goto_sleep](https://user-images.githubusercontent.com/106698115/171481173-fb383525-ec58-4581-8c06-fb011fa9d64c.PNG)
The SX126xCheckDeviceReady() function tries to wakeup the transceiver but the BUSY pin stays high after toggling the NSS pin (i tried different durations from 150 microseconds to 100 milliseconds, with and without a spi transfer after 150 microseconds with the bytes [RADIO_GET_STATUS, 0x00]).
![wakeup_try](https://user-images.githubusercontent.com/106698115/171481552-0ccbd4d8-4f86-4b62-8e80-138ada6819f3.PNG)
The IRQ pin also stays low. Does somebody observed the same behaviour and knows a way to fix it?

**Comment by anson-wa** [comment]
Hi phemidal,

Have you solve this issue?
I am using Mbed OS 5 faced this issue too. The busy pin in high state forever after setSleep.

**Comment by luishrmc** [comment]
HI,

I'm facing the same issue... Has anyone managed to solve this?

**Comment by FatihTS** [comment]
Greetings everyone,

I am experiencing the same problem. I can't understand what's causing this problem. If anyone finds a solution, I look forward to your help. Thanks.

**Reply by domrich76** [reply]
Busy go in HiZ state vhen not busy. So you need add a pulldown resistor or configure the input of µcontroller in INTPUT_PULLDOWN when possible.
DR F1BVV
